apps/web/tests/page-models/authLoginPage.ts

```diff
@@ -0,0 +1,58 @@
+import { Page, selectors } from '@playwright/test';
+import { setBrowserDateTimeTo } from '../utils.ts/browser';
+
+export class LoginData {
+  email: string;
+  password: string;
+}
+
+export class AuthLoginPage {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** nice and clean approach!